### ISSUE TITLE
Default Config Template is slightly wrong for local storage

### ISSUE THREAD

**Issue by hongkongkiwi** [clems4ever/authelia#210]
Hi there,

Very small issue, in the default configuration template, for the local variables it shows that we can do:

```
storage:
  local: '/tmp/storage'
```

but the code has been modified to expect an object, so it should be updated to show

```
storage:
  local:
    path: '/tmp/storage'
```

Small mistake but I think worth fixing.

**Comment by clems4ever** [comment]
Hello @hongkongkiwi ,

Thank you for reporting. I will fix it shortly.

Thanks.

**Comment by clems4ever** [comment]
Hello @hongkongkiwi , I fixed it with #212 .
